test/collection_test.js

```diff
@@ -2171,8 +2175,8 @@ describe("Collection", () => {
     });
 
     it("should validate the remote option", () => {
-      expect(() => articles.sync({remote: "http://fake.invalid"}))
-        .to.Throw(Error, /contain the version/);
+      return articles.sync({remote: "http://fake.invalid"})
+        .should.be.rejectedWith(Error, /contain the version/);
```

Comment:
So this means we're actually breaking the public API; previously we were synchronously throwing, but now that `sync()` is declared entirely async, it's not possible anymore. We should add a note about this in the release changelog.